<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComPort_Setting</class>
 <widget class="QDialog" name="ComPort_Setting">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Port Setting</string>
  </property>
  <widget class="QComboBox" name="portBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="portLable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>32</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>端口号</string>
   </property>
  </widget>
  <widget class="QLabel" name="baudrateLable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>波特率</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DataBitText">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QLabel" name="dataBitLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>数据位</string>
   </property>
  </widget>
  <widget class="QLabel" name="checkBitLable">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>校验位</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="checkBitText">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="stopBitLable">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>停止位</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stopBitText">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QComboBox" name="baudrateBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentText">
    <string>115200</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>230400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>460800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>921600</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
